pkgs/applications/audio/easyabc/default.nix

```diff
@@ -16,8 +16,18 @@ python39.pkgs.buildPythonApplication {
 
   propagatedBuildInputs = with python39.pkgs; [
     cx_Freeze
-    wxPython_4_2
     pygame
+
+    # currently broken with 4.2.1
+    # https://github.com/jwdj/EasyABC/issues/75
+    (wxPython_4_2.overrideAttrs (args: rec {
+      version = "4.2.0";
+      src = fetchPypi {
+        inherit version;
+        pname = "wxPython";
+        hash = "sha256-ZjzrxFCdfl0RNRiGX+J093+VQ0xdV7w4btWNZc7thsc=";
+      };
+    }))
```

Comment:
This should use a python package set overwrite otherwise we might end up with two different wxpython which will break the build.

Reply:
Sorry no idea what a "python package set overwrite" is 😕 

> we might end up with two different wxpython which will break the build.

I don't follow... I _do_ want a different specific version of wxpython here, otherwise easyabc breaks.

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/243787/commits/d373fd420a1265152258e25e03a36368fef6c7b9

Reply:
> I don't follow... I _do_ want a different specific version of wxpython here, otherwise easyabc breaks.

some other dependency could also use it and we want to make sure that we only use one wxpython for everything